packages/web/src/pages/rewards-page/components/modals/ChallengeRewardsModal/CooldownSummaryTable.tsx

```diff
@@ -0,0 +1,55 @@
+import { useCallback } from 'react'
+
+import { useChallengeCooldownSchedule } from '@audius/common/hooks'
+import { ChallengeRewardID } from '@audius/common/models'
+import { Text } from '@audius/harmony'
+
+import { SummaryTable } from 'components/summary-table'
+
+const messages = {
+  upcomingRewards: 'Upcoming Rewards',
+  audio: '$AUDIO'
+}
+
+type CooldownSummaryTableProps = {
+  challengeId: ChallengeRewardID
+}
+
+export const CooldownSummaryTable = ({
+  challengeId
+}: CooldownSummaryTableProps) => {
+  const {
+    cooldownChallenges,
+    summary,
+    isEmpty: isCooldownChallengesEmpty
+  } = useChallengeCooldownSchedule({ challengeId })
+
+  const formatLabel = useCallback((item: any) => {
+    const { label, claimableDate, isClose } = item
```

Comment:
should `isClose` be `isClosed`?

Reply:
same deal, i think the ai may have hallucinated this part. going to see what's currently in main